cg/services/run_devices/run_names/pacbio.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from cg.services.run_devices.run_names.service import RunNamesService
+
+
+class PacbioRunNamesService(RunNamesService):
+
+    def get_run_names(self) -> list[str]:
+        """
+        Get all the run names from the PacBio sequencing directory in the form
+        <sequencing_run>/<SMRTcell>, for example:
+            r84202_20240913_121403/1_C01
+        """
+
+        run_names = []
+        for run_folder in Path(self.run_directory).iterdir():
+            if run_folder.is_dir():
```

Comment:
I do think that this check indicates that we have a trust issue which can cause harm. I have added a discussion point to the coding guidelines. I would let the code break and ensure that the structure is correct if this check fails